test/storage/storages.test.ts

```diff
@@ -81,9 +83,8 @@ describe('General storage functions', () => {
 
     const cache = await axios.storage.get(res1.id, {
       // sample of a request config. Just to the test pass.
-      //@ts-expect-error ignore
       [symbol]: true
-    });
+    } as CacheRequestConfig);
```

Comment:
Having a single `ts-expect-error` comment covering a single field is safer than casting the whole object into the shape you want.

Reply:
OK, proposing a different approach to remove TypeScript error without special handling